CMakeLists.txt

```diff
@@ -2,7 +2,7 @@
 # Licensed under the MIT License.
 
 # Minimum CMake required
-cmake_minimum_required(VERSION 3.25)
+cmake_minimum_required(VERSION 3.28)
```

Comment:
starting from 3.28, cmake supports EXCLUDE_FROM_ALL which already used in some project. A lower version cmake actually doesn't report any issue, which confused us to see some inconsistent build processes without any useful log.